README.md

```diff
@@ -333,17 +333,17 @@ If you are worried or don't know where to start, you can **always** reach out to
 
 We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributor's Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help.
 
-_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!!
+_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to [webpack@opencollective.com](https://opencollective.com/) and we can give pointers or tips!!!
```

Comment:
Link is wrong. It should be a email address.

Reply:
Ok I will make the changes

Reply:
Which email address I have to put here?
When I tried to put "webpack@opencollective.com" it is showing "404 page not found"